app/_landing_pages/gateway/secrets-management.yaml

```diff
@@ -164,6 +164,29 @@ rows:
                 url: "/gateway/entities/vault/?tab=conjur#vault-provider-specific-configuration-parameters"
               - text: With {{ site.base_gateway }}
                 url: "/how-to/configure-cyberark-as-a-vault-backend/"
+  - header:
+      text: AWS Secrets Manager credentials
+      type: h3
+    columns:
+      - blocks:
+        - type: structured_text
+          config:
+            blocks:
+              - type: text
+                text: |
+                  To access secrets stored in the AWS Secrets Manager, {{site.base_gateway}} needs to be configured with an IAM Role that has sufficient permissions to read the required secret values.
+
+                  {{site.base_gateway}} can automatically fetch IAM role credentials based on your AWS environment, observing the following precedence order:
+                  - Fetch from credentials defined in environment variables `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
+                  - Fetch from profile and credential file, defined by `AWS_PROFILE` and `AWS_SHARED_CREDENTIALS_FILE`.
+                  - Fetch from an ECS [container credential provider](https://docs.aws.amazon.com/sdkref/latest/guide/feature-container-credentials.html).
+                  - Fetch from an EKS [IAM roles for service account](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).
+                  - Fetch from EC2 IMDS metadata. Both v1 and v2 are supported
+
+                  {{site.base_gateway}} also supports role assuming (with [`vaults.config.assume_role_arn` and `vaults.config.role_session_name`](/gateway/entities/vault/?tab=aws#vault-provider-specific-configuration-parameters))which allows you to use a different IAM role to fetch secrets from AWS Secrets Manager.  This is a common practice in permission division and governance and cross-AWS account management.
+
+                  {:.info}
+                  > **Note:** IAM Identity Center credential provider and Process credential provider are not supported.
   - header:
       text: Secrets rotation
       type: h2
```

Comment:
I think I would put this lower on the page since its so provider specific. I feel like the landing page should be a bit more general and then point to the specific thing.